integration-tests/src/test/resources/features/processor.feature

```diff
@@ -30,3 +31,92 @@ Feature: Processor tests
     And the Manager metric 'manager_processor_status_change_total{status="PROVISIONING",}' count is at least 1
     And the Manager metric 'manager_processor_status_change_total{status="READY",}' count is at least 1
     And the Manager metric 'manager_processor_status_change_total{status="DELETED",}' count is at least 1
+
+  Scenario: Processor filter is updated
+    Given add a Processor to the Bridge "mybridge" with body:
+    """
+    {
+      "name": "myProcessor",
+      "action": {
+        "parameters": {
+            "topic":  "myKafkaTopic"
+        },
+        "type": "KafkaTopic"
+      }
+    }
+    """
+    And the Processor "myProcessor" of the Bridge "mybridge" is existing with status "ready" within 3 minutes
+
+    When update the Processor "myProcessor" of the Bridge "mybridge" with body:
+    """
+    {
+      "name": "myProcessor",
+      "action": {
+        "parameters": {
+            "topic":  "myKafkaTopic"
+        },
+        "type": "KafkaTopic"
+      },
+      "filters": [
+       {
+        "type": "StringEquals",
+        "key": "source",
+        "value": "StorageService"
+       }
+     ]
+    }
+    """
+    And the Processor "myProcessor" of the Bridge "mybridge" is existing with status "ready" within 3 minutes
+
+    When update the Processor "myProcessor" of the Bridge "mybridge" with body:
+    """
+    {
+      "name": "myProcessor",
+      "action": {
+        "parameters": {
+            "topic":  "myKafkaTopic"
+        },
+        "type": "KafkaTopic"
+      },
+      "filters": [
+       {
+        "type": "StringEquals",
+        "key": "source",
+        "value": "StorageService"
+        },
+        {
+        "type": "ValuesIn",
+        "key": "data.any",
+        "values": ["John", 2]
+        },
+        {
+        "type": "StringBeginsWith",
+        "key": "data.name",
+        "values": ["John", "Mar"]
+        },
+        {
+        "type": "StringContains",
+         "key": "data.name",
+           "values": [
+              "John",
+              "Marco"
+            ]
+        }
+     ]
+    }
+    """
+    And the Processor "myProcessor" of the Bridge "mybridge" is existing with status "ready" within 3 minutes
+
+    When update the Processor "myProcessor" of the Bridge "mybridge" with body:
+    """
+    {
+      "name": "myProcessor",
+      "action": {
+        "parameters": {
+            "topic":  "myKafkaTopic"
+        },
+        "type": "KafkaTopic"
+      }
+    }
+    """
+    And the Processor "myProcessor" of the Bridge "mybridge" is existing with status "ready" within 3 minutes
```

Comment:
Can you please send a cloud event to check that the updated processor is actually deployed and updated filtering is applied?

Reply:
Sure will add a step to send cloud events after filter update